example/src/components/buttons/PinButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { AdditionalColors, TextColors } from '@colors';
+import { Icon } from '@components/Icon';
+import { Typo } from '@components/Typo';
+import React, { ReactNode } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+type PinButtonProp = {
+  onPress: () => void;
+  children: ReactNode;
+};
+
+export const PinButton = ({ children, onPress }: PinButtonProp) => {
+  return (
+    <TouchableOpacity onPress={onPress} style={styles.container}>
```

Comment:
when I click pin button now the touchable opacity is not rounded:

https://user-images.githubusercontent.com/6280457/229092377-efec624d-e375-4a38-8977-543f8c0c48ae.mp4

Reply:
Which emulator have you used? I don't see such a behaviour on mine.

Reply:
I used my real android device

Reply:
Should be fixed now. But please double check.